src/__tests__/Button.test.js

```diff
@@ -88,16 +92,22 @@ describe("<Button />", () => {
   });
 
   it('color="#88B04B"', () => {
-    const component = renderer.create(<Button color="#88B04B" />).toJSON();
-    const instance = renderer.create(<Button color="#88B04B" />).getInstance();
+    const B = renderer.create(<Button color="#88B04B" />);
```

Comment:
I like then variable names are self explanatory. B could be Button but it conflicts with the component name, then we can name it something more relevant, maybe: button or something else :)